src/CSI.IBTA.Administrator/Services/JwtTokenService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.IdentityModel.Tokens.Jwt;
+using CSI.IBTA.Administrator.Interfaces;
+using CSI.IBTA.Shared.Entities;
+
+namespace CSI.IBTA.Administrator.Services
+{
+    //should this class be moved to Shared project so that it could be reused by Employer and Consumer portals?
+    public class JwtTokenService : IJwtTokenService
+    {
+        private readonly ILogger<JwtTokenService> _logger;
+        public JwtTokenService(ILogger<JwtTokenService> logger)
+        {
+            _logger = logger;
+        }
+
+        public CookieOptions GetCookieOptions()
+        {
+            return new CookieOptions
+            {
+                HttpOnly = true
+            };
+        }
+
+        public async Task<(bool isAdmin, string token)> CheckUserIsAdminAsync(HttpResponseMessage response)
```

Comment:
```suggestion
        public async Task<(bool isAdmin, string token)> IsAdminAsync(JToken token)
```

Reply:
Lets pass token as a param, not the http response. The class is supposed to work with tokens and it should not matter how it was received - via http call, DB or from memory cache. 

Extract token from http in the controller.